### DISCUSSION TITLE
Storage of metrics retrieved with KPIMON xAPP

### DISCUSSION THREAD

**Discussion by joseluiscarcel** [srsran/oran-sc-ric#53] [General]
Good afternoon! 

I have successfully deployed the i-release RIC and the KPIMON xApp together with the srsRAN 24.10 and I wanted to start to analyze the metrics retrieved. In this context, I wanted to ask if there is any mechanism implemented that allows the storage of the metrics retrieved by the KPIMONxApp from the srsRAN (apart from printing the metrics in the screen).

If the answer is NO, could the dbaas be reused to store these metrics or is it better to try to deploy an influxdb container as in other versions of the oran-sc-ric?

Thanks in advance.

Regards,
José Luis

**Comment by joseluiscarcel** [comment]
@pgawlowicz any idea about this? Thanks!

**Reply by pgawlowicz** [reply]
Hi, I cannot help regarding the usage of dbaas. I think that the content of RIC indication is not stored there.
But an xApp is simply Python, so it should be easy to connect to any database (or even collect the values and save to a file).
